environment_setup/tools/macos_fw.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/WORKSPACE b/src/WORKSPACE
+index a18004d..cfaf0d9 100644
+--- a/src/WORKSPACE
++++ b/src/WORKSPACE
+@@ -97,10 +97,10 @@ http_archive(
```

Comment:
Good to know what needs to get modified here, we definitely need to implement this in a more stable form (ie. not patching :joy: )